.github/workflows/cd.yml

```diff
@@ -20,13 +23,18 @@ jobs:
         run: pip install poetry
       - name: Install dependencies
         run: poetry install
+      - name: Run pylint
+        run: poetry run pylint src
```

Comment:
This fails the whole job if there is a single warning. While we should strive for a perfect score, and warnings can be suppressed if really necessary, it might be too limiting.

Consider adding e.g. `--fail-under=8`

Reply:
added